.github/pull_request_template.md

```diff
@@ -0,0 +1,13 @@
+Soundtrack of this PR: [link to song that really fits the mood of this PR]()
```

Comment:
Buzzkill mode on: I'm going to figure out how to embed an auto-play "Scorpions: Wind of Change" here until everyone has that goddamned whistle stuck in their head, because the "sound track" of my PR is the same as everyone else: whatever trash EDM I was playing to drown out my upstairs neighbors' pack of barking dogs.

Reply:
hahahahaha